src/util/humanizeByteCount.test.ts

```diff
@@ -0,0 +1,41 @@
+import humanizeByteCount from './humanizeByteCount';
+
+describe('humanizeByteCount', () => {
+  it.each([[0, 0], [0, 3], [0, -1]])(
+    'handles 0 bytes accordingly',
+    (byteCount, decimalCount) => {
+      expect(humanizeByteCount(byteCount, decimalCount)).toBe('0 B');
+    },
+  );
+
+  it.each([[5, 0], [5, 5], [5, -1], [5, undefined], [5, 10000000]])(
+    'handles an exact number of bytes and shows no decimal places',
+    (byteCount, decimalCount) => {
+      expect(humanizeByteCount(byteCount, decimalCount)).toBe('5 B');
+    },
+  );
+
+  it.each([[1048576, 0], [1048576, 2], [1048576, undefined], [1048576, 5]])(
+    'converts bytes to megabytes and shows no decimal point, even when we request it',
+    (byteCount, decimalCount) => {
+      expect(humanizeByteCount(byteCount, decimalCount)).toBe('1 MB');
+    },
+  );
+
+  it('handles decimal point accordingly when the conversion value is not an exact number', () => {
+    expect(humanizeByteCount(340000, 0)).toBe('332 KB');
```

Comment:
You can jest.each these too, just add a third parameter for the expected string.